tests/end_to_end/empty-if/manual-out.c

```diff
@@ -1,5 +1,5 @@
 void test(s32 arg0, s32 arg1) {
-    if ((arg0 == 1) || (arg1 != 2)) {
```

Comment:
This *looks* bad but the underlying assembly is actually ambiguous -- either is fine. (The if block is empty)